.flake8

```diff
@@ -58,6 +59,9 @@ ignore =
     D402
     # First word of the first line should be properly capitalized
     D403
+
+    # Annotation in typing.cast() should be a string literal
+    TC006
```

Comment:
I will probably do this separately (it helps with runtime performance, though it's unclear how noticeable is the difference, and more importantly it will forbid non-string `Deferred[foo]` expressions which aren't supported on older Twisted).